.circleci/workflows.yml

```diff
@@ -333,22 +333,6 @@ jobs:
           condition: *validate-sql
           steps:
             - checkout
-            - run:
-                name: Early return when job not modified
-                command: |
-                  if [ "$CIRCLE_BRANCH" = main ]; then
-                    echo "Run job because branch is $CIRCLE_BRANCH"
-                  elif git log --format=%B --no-merges -n 1 |
-                      grep -qF '[run-tests]'; then
-                    echo "Run job because [run-tests] in commit message"
-                  elif ! git diff --quiet origin/main... \
```

Comment:
Would it make sense to compare against the dags in the generated-sql branch instead?  Probably doesn't matter too much since I don't think validate-dags is a bottleneck and I would prefer to have fewer conditions hidden in the CI

Reply:
Yeah, I also prefer simplifying the CI a bit. If it turns out that this slows things down too much then we can add more skip logic